src/transition-wrapper/TransitionWrapper.d.ts

```diff
@@ -0,0 +1,29 @@
+import { CSSProperties, PureComponent } from 'react';
+import { Type } from 'tcomb';
+
+export namespace TransitionWrapperProps {
+  type TransitionStyles = {
+    enter?: CSSProperties,
+    enterActive?: CSSProperties,
+    default?: CSSProperties,
+    leave?: CSSProperties,
+    leaveActive?: CSSProperties
+  }
+}
+
+export type TransitionWrapperProps = {
+  children: any, // TODO: t.ReactChildren
+  component: React.ComponentClass | string,
```

Comment:
ComponentType should be better (otherwise it wouldn't accept stateless functional components.
And, as you told me [here](https://github.com/buildo/react-components/pull/960#discussion_r125475584), there should be a type parameter